packages/backend/src/accounting/transfers.ts

```diff
@@ -1,39 +1,35 @@
 import {
-  Commit,
-  CommitFlags,
-  CommitTransferError as CommitTransferErrorCode,
   CreateTransferError as CreateTransferErrorCode,
   Transfer as TbTransfer,
   TransferFlags
 } from 'tigerbeetle-node'
 import { v4 as uuid } from 'uuid'
 
-import {
-  CommitTransferError,
-  CreateTransferError,
-  TransferError
-} from './errors'
+import { CreateTransferError, TransferError } from './errors'
 import { ServiceDependencies } from './service'
 import { AccountId, toTigerbeetleId } from './utils'
 
-const TRANSFER_RESERVED = Buffer.alloc(32)
+const ACCOUNT_TYPE = 1
 
 type TransfersError = {
   index: number
   error: TransferError
 }
 
 export interface CreateTransferOptions {
-  id?: string
+  id?: string | bigint
   sourceAccountId: AccountId
   destinationAccountId: AccountId
   amount: bigint
+  ledger: number
   timeout?: bigint
+  pendingId?: string | bigint
 }
 
 export async function createTransfers(
   deps: ServiceDependencies,
-  transfers: CreateTransferOptions[]
+  transfers: CreateTransferOptions[],
+  commit?: boolean
```

Comment:
```suggestion
  transfers: CreateTransferOptions[]
```
What we remove `commit` and replace `pendingId` with optional `commitId` and `rollbackId`
You can make them mutually exclusive by removing `pendingId` renaming `CreateTransferOptions` to `CreateTransferOptionsBase` and adding something like:
```
export interface CommitTransferOptions extends CreateTransferOptionsBase {
  commitId?: string | bigint
  rollbackId?: never
}

export interface RollbackTransferOptions extends CreateTransferOptionsBase {
  commitId?: never
  rollbackId?: string | bigint
}

export type CreateTransferOptions = CommitTransferOptions | RollbackTransferOptions
```

Reply:
Refer to: https://github.com/interledger/rafiki/issues/518